src/components/accessibility/screen_reader_live/screen_reader_live.tsx

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, {
+  AriaAttributes,
+  HTMLAttributes,
+  FunctionComponent,
+  ReactNode,
+  useEffect,
+  useState,
+} from 'react';
+
+import { EuiScreenReaderOnly } from '../screen_reader_only';
+
+export interface EuiScreenReaderLiveProps {
+  /**
+   * Whether the make screen readers aware of the content
+   */
+  isActive?: boolean;
```

Comment:
Do we think this `isActive` flag will be useful/used by many consumers, especially now that `aria-live="off"` is an option? 🤔 Just thinking out loud, definitely not a blocking/firm change request

Reply:
Oh, super minor typo suggestion if we do still want to go with this prop
```suggestion
  /**
   * Whether to make screen readers aware of the content
   */
  isActive?: boolean;
```